README.md

```diff
@@ -17,33 +17,30 @@ separate Charm.
 The Prometheus Operator may be deployed using the Juju command line as
 in
 
-    juju deploy prometheus-k8s
+    $ juju deploy prometheus-k8s
```

Comment:
You should enclose this in a code block like:
````
```sh
$ juju deploy prometheus-k8s
```
````

Reply:
Fixed.